_posts/en/posts/2016-02-26-zkcp.md

```diff
@@ -0,0 +1,140 @@
+---
+type: posts
+layout: post
+lang: en
+id: en-zkcp-announce
+name: zkcp-announce
+title: The first successful Zero-Knowledge Contingent Payment
+permalink: /en/2016/02/26/zkcp-announce/
```

Comment:
Could we possibly use a more descriptive permalink like `/en/2016/02/26/zero-knowledge-contingent-payments-announcement/` ?